project1/project1/views.py

```diff
@@ -1,11 +1,14 @@
 from django.shortcuts import render, redirect
 from django.utils.translation import activate
-from store.models import Product
+from store.models import Product, Category
 
 
 def home(request):
     products = Product.objects.all().filter(is_available=True)
+    categories = Category.objects.filter(parent=None)
```

Comment:
nên đặt tên biến gần với ý nghĩa của nó nhé e
```suggestion
    parent_categories = Category.objects.filter(parent=None)
```